src/s3/explorer/s3FileNode.ts

```diff
@@ -92,16 +91,3 @@ export class S3FileNode extends AWSTreeNodeBase implements AWSResourceNode {
 function formatBytes(numBytes: number): string {
     return bytes(numBytes, { unitSeparator: ' ', decimalPlaces: 0 })
 }
-
-function fileIconPath(): vscode.ThemeIcon | { light: vscode.Uri; dark: vscode.Uri } {
```

Comment:
Thanks for noticing this duplication and factoring it out!

Reply:
Should I move these functions to this `src/test/shared/utilities/iconPathUtils.ts` file then?

Reply:
Moved to src/shared/utilities/vsCodeUtils